packages/calling/src/CallingClient/CallingClient.test.ts

```diff
@@ -119,7 +119,7 @@ describe('CallingClient Tests', () => {
         callingClient = await createClient(webex, {serviceData: serviceDataObj});
       } catch (e) {
         expect(e.message).toEqual(
-          'Invalid service indicator, Allowed values are: calling,contactcenter'
+          'Invalid service indicator, Allowed values are: calling,contactcenter,guestcalling'
```

Comment:
Please change the code that generates the error too
```suggestion
          'Invalid service indicator, Allowed values are: calling, contactcenter and guestcalling'
```

Reply:
Updated.